### PR TITLE
Scaffold & style front page

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+get_header();
+
+$splash_text = IGV_get_option('_igv_splash_text');
+$splash_text_en = IGV_get_option('_igv_splash_text_en');
+
```
Comment (mikeyrayvon) [inline front-page.php:6]: could this come from the wordpress tagline instead?

File: lib/theme-options.php [hunk 2]
```diff
@@ -133,15 +133,15 @@ function add_options_page_metabox() {
 			'desc' => __( '...', 'cmb2' ),
 			'id'   => $this->prefix . 'splash_text',
 			'type' => 'text',
-			'default' => 'es una plataforma cultural y artística auspiciada por Casa Luis Barragán con el apoyo de Colección FEMSA.',
+			'default' => 'ESTANCIA FEMSA es una plataforma cultural y artística auspiciada por Casa Luis Barragán con el apoyo de Colección FEMSA.',
 		) );
 
 		$options_metabox->add_field( array(
 			'name' => __( 'Splash texto [EN]', 'cmb2' ),
```
Comment (mikeyrayvon) [inline lib/theme-options.php:140]: Texto del splash?

File: front-page.php (hunk 1 above)
Comment (varffs) [inline front-page.php:6]: Possibly but already existing in the previous WP like this. How does tagline deal with language and q-translate?

File: lib/theme-options.php (hunk 2 above)
Comment (varffs) [inline lib/theme-options.php:140]: @mixcas ?

File: front-page.php (hunk 1 above)
Comment (mikeyrayvon) [inline front-page.php:6]: it translates. there are switching buttons

Comment (varffs) [inline front-page.php:6]: I'm pretty happy to leave how is really. It all goes down in the site options page and makes it less complicated for end user

File: lib/theme-options.php (hunk 2 above)
Comment (mixcas) [reply lib/theme-options.php:140]: `Texto del Splash` esta bien

**Pull request merged**